### DISCUSSION TITLE
Wireless Redstone

### DISCUSSION THREAD

**Discussion by ablac** [Zrips/CMI#3512] [Feature Requests]
I would love to see the ability to make wireless redstone in CMI. There used to be an amazing plugin called Wireless Redstone but its no longer being made, and I loved that plugin.

Super simple create a sign.
Create a Transmitter
1.  [WRT]
2. Number (This is the channel)

Create a Receiver sign
1. [WRR]
2. Number (Channel number to listen to)

I believe this would be a great addition to CMI, and would make people that do lots of redstone work over large builds much easier!

**Comment by CloudeLecaw** [comment]
You forgot the timers and repeaters.. ;) 

Well this is exact what the plugin [wireless redstone](https://www.spigotmc.org/resources/wirelessredstone.8251/?__cf_chl_jschl_tk__=ae8b572d6a80100e8f346bbac3a90292f12b33c0-1592291181-0-AdZDQwu62QX-4IPLXvjgzqmLErpY9jWnFvtzJFDYv9lkHRJaxEWiHopzwfKRUva0rKPNGHP5Zc0JI3mYuIG-tbb3CFn6o04yebU1_GJuk22AAhosc-arCfgSlkW0Gt_evTgkVdRXu7zQYb7V4lPZ4EPR7zj34UGtSAJGBnND5Wr2HhkLa_S1Gp0ZRnDXYRB21WMhyGnE6F8Darqi9ZhXXVglJfD1KhwGNuj3arnT2OVwe8siZGTrM6pCAC__GQGpUMkPOGvxCBOQfYu4SZpURjLocDLb_c8FenyAMUAEQlHOue3ymqO_tGRAHtheclRpog)
Why not also add the whole plugin wireless hoper to mange player having lesser hoppers that would be good for all server with big hopper useage. 

 it would be a bit risky because with way like this trolls begin to add there own receivers and taking others channel signals. 

Why not implement also the redstone proxmity sensors? Also a nice solution and often better then the sign based solutions ;) 

 i think this is a bit a way to far from the target that cmi has and should follow.. CMI is no garbage collection of feature of old plugins where the dev decided not to maintain the plugin anymore, to take over into CMI

But finally its up to Zrips the decide if he really wanne implement the whole plugin as cmi Feature.

**Comment by ablac** [comment]
I highly doubt he would implement it without permissions FYI, but I’m not saying to implement it as a garbage dump, I I just used wireless Redstone as an example. Because there’s Literally only one wireless Redstone plugin that’s still getting updated, and it’s terrible, yet it has tens of thousands of downloads. I know Zrips can do it right.

The other thing with wireless Redstone is if your going long distance there’s Thousands of Block updates just in a 3 block Redstone wire. Using wireless redstone reduces those massive amounts of block updates.

And if not signs then maybe attach it to levels, buttons, pressure plates and just toggle them wirelessly. There’s tons of ways to implement it.

**Comment by CloudeLecaw** [comment]
Hm if i use a redstone wire then uts over after sime blocks bcause the chunks are not loaded anymore ;)
